.travis.yml

```diff
@@ -45,6 +45,9 @@ jobs:
     - stage: compile
       name: "ICL Build"
       script: ./scripts/docker-run.sh ./scripts/xtensa-build-all.sh -l icl
+    - stage: compile
+      name: "IMX Build"
+      script: ./scripts/docker-run.sh ./scripts/xtensa-build-all.sh -l imx
```

Comment:
name: "IMX8 Build"
And the last argument will be -l imx8.

Reply:
Got it.